### PR TITLE
Update rubygem-foreman_remote_execution to 3.0.1

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
This is now on hold, we need to update gemspec to also accept tasks 1.0.x, expect 3.0.1 to fix that

**Comment by ares** [comment]
Updated to 3.0.1, requirement for tasks < 1 was dropped

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -53,23 +51,15 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
...
+BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.3.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 4.0.0
 # end package.json devDependencies BuildRequires
 
 # start package.json dependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(@theforeman/vendor) >= 1.4.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/vendor) < 2.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/vendor) >= 3.3.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/vendor) < 4.0.0
```
Comment (tbrisker) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: This will fail to build now, please update

Comment (ares) [reply packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: this is coming from the bump_rpm.sh, should I change that manually? I'd expect bump_rpm to pick the right version after https://github.com/theforeman/foreman_remote_execution/commit/8fc4446e4bd7ba5f56ba2cc639b0afa01021f5f9 got in

Comment (ares) [reply packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: oh, please ignore me, we need to release, I though v4 was already in 3.0.1

**Comment by adamruzicka** [comment]
This is now contained in https://github.com/theforeman/foreman-packaging/pull/4621

**Comment by tbrisker** [comment]
closing for #4621

**Pull request closed without merging**